public/sass/pages/_dashboard.scss

```diff
@@ -68,11 +68,17 @@ div.flot-text {
   font-weight: $font-weight-semi-bold;
   position: relative;
   width: 100%;
-  display: block;
-  padding-bottom: 2px;
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: center;
+  padding: 4px 0 4px;
 }
 
 .panel-title-text {
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+  max-width: calc(100% - 28px);
```

Comment:
Changing to 48px or similar pushes the down arrow icon more to the left and adds some space to the right - I think it looks better - what do you think?